lib/utilities/errors/crash_handler.dart

```diff
@@ -0,0 +1,127 @@
+import 'dart:async';
+
+import 'package:firebase_core/firebase_core.dart';
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:mm_flutter_app/utilities/debug_logger.dart';
+import 'package:mm_flutter_app/utilities/errors/error_widget.dart';
+import 'package:mm_flutter_app/utilities/errors/errors.dart';
+import 'package:mm_flutter_app/utilities/errors/exceptions.dart';
+import 'package:retry/retry.dart';
+
+class CrashHandler {
+  bool _isReleaseMode = false;
+  bool _isCollectionEnabled = false;
+
+  CrashHandler(bool isReleaseMode, bool isCollectionEnabled) {
+    _isReleaseMode = isReleaseMode;
+    _isCollectionEnabled = isCollectionEnabled;
+    _initialize();
+  }
+
+  void _initialize() async {
+    await Firebase.initializeApp();
```

Comment:
I pulled from this branch but failed at this step with web, fixed by changing it to `Firebase.initializeApp(options: DefaultFirebaseOptions.currentPlatform);`.

Reply:
Ah very nice catch! I just add a commit to fix this. Thanks!